Update manual deployment guide of Scalar DB (#187)
scalar-labs/scalar-kubernetes: docs/ManualDeploymentGuideScalarDBServerOnEKS.md

* [How to install Scalar products through AWS Marketplace](./AwsMarketplaceGuide.md)

Note: Please see the **Subscribe to Scalar products from AWS Marketplace** section in the above document.  

## Step 2. Set up a database for Scalar DB Server

Scalar DB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy Scalar DB Server.  

Please refer to the following document for more details.  

* [Set up a database for Scalar DB/Scalar DL deployment on AWS](./SetupDatabaseForAWS.md) // TODO: Update existing document

## Step 3. Create EKS Cluster

Create EKS Cluster for the deployment of Scalar DB Server. Please refer to the following document for more details.  

* [Create EKS Cluster for Scalar Products]() // TODO: Create a new document

## Step 4. Create bastion server

For executing some tools to deploy and manage Scalar DB Server on EKS, you need to prepare a bastion server in the same VPC of the EKS Cluster you created in **Step 3**. Please refer to the following document for more details.  

* [Create bastion server]() // TODO: Create a new document

> ## Step 5. Prepare custom values file of Helm

Reviewer: ```suggestion
## Step 5. Prepare a custom values file of Helm
```

You need to configure your custom values file for Helm Chart of Scalar DB Server based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  

* [Configure custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository

## Step 6. Deploy Scalar DB Server using Scalar Helm Chart

Deploy Scalar DB Server on your EKS Cluster using Scalar Helm Chart. Please refer to the following document for more details.  

* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository

## Step 7. Check the status of Scalar DB Server deployment

After deploying Scalar DB Server on your EKS Cluster, you need to check the status of each component. Please refer to the following document for more details.  

* [What you might want to check on a regular basis](./RegularCheck.md) // TODO: Update existing document

## Step 8. Monitoring for Scalar DB Server deployment

> After deploying Scalar DB Server on your EKS Cluster, you need to monitor each component and collect logs. Please refer to the following document for more details.  

Reviewer: ```suggestion
After deploying Scalar DB Server on your EKS Cluster, it is recommended to monitor the deployed components and collect their logs, especially in production. Please refer to the following document for more details.  
```

* [Kubernetes Monitor Guide](./K8sMonitorGuide.md) // TODO: Update existing document
* [How to collect logs from Kubernetes applications](./K8sLogCollectionGuide.md) // TODO: Update existing document

---

## Uninstall Scalar DB Server on EKS

> If you want to uninstall the environment you created in the above steps, please uninstall/remove resources in the reverse order of creation.  // TODO: Add delete steps in each document

Reviewer: ```suggestion
If you want to uninstall the environment you created, please uninstall/remove resources in the reverse order of creation.  // TODO: Add delete steps in each document
```
